pkg/printers/internalversion/printers.go

```diff
@@ -1151,6 +1152,25 @@ func getNodeExternalIP(node *api.Node) string {
 	return "<none>"
 }
 
+// Returns the internal IP of the node or "<none>" if none is found.
+// If multiple internal IPs are found, ",+more" are appended after the first one.
+func getNodeInternalIP(node *api.Node) string {
+	var ips []string
+	for _, address := range node.Status.Addresses {
+		if address.Type == api.NodeInternalIP {
+			ips = append(ips, address.Address)
+		}
+	}
+
+	ipsNum := len(ips)
+	if ipsNum > 1 {
+		return ips[0] + fmt.Sprintf(",+%dmore", ipsNum-1)
```

Comment:
Please put space between number and `more`

Reply:
Discussed with @dixudx on slack, this might cause additional problems. I'm ok with this as is for now.